server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowTaskInstanceService.java

```diff
@@ -214,7 +216,7 @@ public List<? extends FlowTaskResult> getResult(@NotNull Long id) throws IOExcep
             return getAsyncResult(taskEntity);
         } else if (taskEntity.getTaskType() == TaskType.MOCKDATA) {
             return getMockDataResult(taskEntity);
-        } else if (taskEntity.getTaskType() == TaskType.EXPORT || taskEntity.getTaskType() == TaskType.IMPORT) {
```

Comment:
how about export task?

Reply:
Export task was processed above in line#206